apps/sync/test/unit/crypto/stringconversion_test.js

```diff
@@ -85,6 +99,22 @@ suite('utils', () => {
           throw(Error);
     });
   });
+  suite('byteArrayToUtf16String', () => {
+    test('converts an Uint8Array to a UTF-16 string', () => {
+      var array = new Uint8Array(3);
+      array[0] = 0xe2;
+      array[1] = 0x82;
+      array[2] = 0xac;
```

Comment:
Yes, that would save a lot of confusion. We don't care how String objects are represented internally, let's call all Strings just Strings, and call all ByteArrays Utf8ByteArray so that it's clearer.